Set-SSLSecurity.ps1

```diff
@@ -107,7 +107,17 @@ process {
     New-WebBinding -Name ChocolateyCentralManagement -Protocol https -Port 443 -SslFlags 0 -IpAddress '*'
 
     #Start the components back up
-    Start-Website ChocolateyCentralManagement
+    try {
+        Start-Website ChocolateyCentralManagement
+    }
+    catch {
+        #Try again...harder.
+        Write-Warning "Unable to start Chocolatey Central Management website, retrying"
+        Get-Website ChocolateyCentralManagement | Start-Website -ErrorAction Stop
```

Comment:
```suggestion
        Get-Website ChocolateyCentralManagement | Start-Website
        Start-Sleep -Seconds 5
        Get-Website ChocolateyCentralManagement | Start-Website -ErrorAction Stop
```

I don't know if it's worth adding a third try, with a wait/sleep?

Reply:
Not sure either. My gut says trying twice will be good enough. However, We could try/catch the same code, then do it by pipeline in the finally, with a warning if that also fails.

Reply:
I'm good with that!

If you put up a commit with those changes, I'll go ahead and approve.

Please and thank you! 🙏🏼 ❤️

Reply:
Try/Catch/Finally isn't gonna work here. I forgot. Finally will fire even after a successful Try {} block. Jig it slightly differently, I shall.